core/src/test/scala/unit/kafka/coordinator/transaction/TransactionMarkerChannelManagerTest.scala

```diff
@@ -59,13 +60,15 @@ class TransactionMarkerChannelManagerTest {
     reaperEnabled = false)
   private val time = new MockTime
 
+  private val metrics = new Metrics()
```

Comment:
This is just a test. We often just create a Metrics instance like this. It's important that we close it as a thread is created for it.